pages/inventories/keycaps/index.js

```diff
@@ -66,15 +100,39 @@ export default function Keycaps() {
       <StyledContainer>
         <h1>Keycap Inventory</h1>
         {data?.length ? (
-          <InventoryCard data={data} />
+          isEditMode ? (
+            <AttentionSeeker effect="shake">
+              <InventoryCard
+                data={data}
+                isEditMode={isEditMode}
+                onDelete={handleDeleteKeycap}
+              />
+            </AttentionSeeker>
+          ) : (
+            <InventoryCard
+              data={data}
+              isEditMode={isEditMode}
+              onDelete={handleDeleteKeycap}
+            />
+          )
         ) : (
           <>
-            <p> You have no keycaps in your inventory!</p>
-            <p> Click the ➕ button to add a keycap set</p>
+            <p>You have no keycaps in your inventory!</p>
+            <p>Click the ➕ button to add a keycap set</p>
           </>
         )}
       </StyledContainer>
       <AddButton onOpenModal={() => setIsOpen(true)} />
+      <EditInventoryButton
+        isEditMode={isEditMode}
+        onToggleEdit={() => {
+          if (isEditMode) {
+            setIsEditMode(false);
+          } else {
+            setIsEditMode(true);
+          }
+        }}
```

Comment:
You can simplify this to:

```suggestion
        onToggleEdit={() => setIsEditMode((prevMode) => !prevMode)}
```

Reply:
Implemented, and done!